modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenSecurity.java

```diff
@@ -27,7 +27,7 @@ public class CodegenSecurity {
     public String name;
     public String type;
     public String scheme;
-    public Boolean isBasic, isOAuth, isApiKey;
+    public Boolean isBasic, isOAuth, isApiKey, isOIDC;
```

Comment:
the PR is mixing OIDC (shortcut) and openIdConnect (full name)
should it be unified? @wing328

Reply:
Personally I prefer the full name.

Reply:
Used fulll name everywhere.